Dockerfile

```diff
@@ -3,6 +3,8 @@ LABEL maintainer="SkynetLabs <devs@skynetlabs.com>"
 
 WORKDIR /root
 
+ENV CGO_ENABLED=0
```

Comment:
kind of weird you needed this even though we originally built form golang alpine, I thought you only needed this if it was debian based